README.md

```diff
@@ -30,30 +30,30 @@
 
 ## What is DevChat?
 
-DevChat is an open-source platform that empowers developers to leverage AI for code generation and documentation. We aim to go beyond simple code auto-completion and limited operations on code snippets. DevChat offers a highly *practical* and *effective* way for developers to interact and collaborate with large language models (LLMs).
+DevChat is an open-source platform that empowers developers to more naturally and effectively integrate AI into code generation and documentation. DevChat aims to go beyond simple auto-completion and limited operations on code snippets. DevChat offers a highly *practical* and *effective* way for developers to interact and collaborate with large language models (LLMs), DevChat is the tool that works for you, no need to change your workflow.
 
 ## Why DevChat?
 
-While there are many AI coding tools available, we developed DevChat based on our practical insights from generating tens of thousands of lines of code. DevChat makes the following distinctive design choices:
+We spent years on code analysis, static-analysis, and DevOps improvements, working with hundreds of companies around the world. The insights we have picked up in this process are all incorporated into DevChat. The techniques, approaches, and decisions that drive success for companies large and small inform every detail of DevChat. You'll find some distinctive design choices:
 
-- **Precise manual control over the context embedded in a prompt**. Precise control over context is the key to effective AI use. We find that most other "intelligent" or "automatic" tools tend to over-guess what a user needs to put into a prompt. That typically introduces more noise than LLMs can effectively manage.
-- **A simple, extensible prompt directory**. This enables developers or teams to easily integrate their own prompt templates into DevChat, avoiding significant engineering effort or a steep learning curve. You don't need a complex framework to make AI work for you. All it takes is a standard editor operating on your filesystem.
+- **Precise manual control over the context embedded in a prompt**. DevChat won't create new work for you, because it ensures that context drives everything. Precise control over context is the key to effective AI use. We find that most other "intelligent" or "automatic" tools tend to over-guess what a user needs to put into a prompt. That typically introduces more noise than LLMs can effectively manage. 
```

Comment:
The point is not that the context drives everything. We actually require humans do a little bit more work in specifying the right context but makes AI much more effective as return.